pkg/spanner/client.go

```diff
@@ -211,18 +212,34 @@ func (c *Client) ApplyDDL(ctx context.Context, statements []string) error {
 	return nil
 }
 
-func (c *Client) ApplyDMLFile(ctx context.Context, ddl []byte, partitioned bool) (int64, error) {
+type PriorityType int
+
+const (
+	PriorityTypeHigh PriorityType = iota
```

Comment:
since the zero value for `int` is `0`, this should be `iota + 1` to distinguish the invalid value.

Reply:
fixed